Core/Core/View/Base/Webview/Models/ColorInvertionInjection.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ColorInvertionInjection.swift
+//  Core
+//
+//  Created by Vadim Kuznetsov on 31.01.24.
+//
+
+import WebKit
+
+public struct ColorInvertionInjection: WebViewScriptInjectionProtocol, CSSInjectionProtocol {
```

Comment:
minor nit: Invertion isn't a valid word, I'd suggest renaming the class to `ColorInversionInjection`

Reply:
renamed